src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.cs

```diff
@@ -99,5 +102,50 @@ public LambdaComparer(Func<T?, T?, bool> equal, Func<T?, int> getHashCode)
 
             public int GetHashCode(T obj) => _getHashCode(obj);
         }
+
+        private const string RegexGeneratorAttributeSource =
+@"#nullable enable
+using global::System.Diagnostics;
```

Comment:
NIT: how come we are using `global::` to qualify usings but not the rest of the System.* types below? I suppose we would need to also use global bellow in case people are for whatever reason declaring types that clash with some of ours?